easybuild/easyconfigs/c/crb-blast/crb-blast-0.6.9-intel-2017a-Ruby-2.3.4.eb

```diff
@@ -0,0 +1,56 @@
+easyblock = 'Bundle'
+
+name = 'crb-blast'
+version = '0.6.9'
+
+rubyver = '2.3.4'
+versionsuffix = '-Ruby-%s' % rubyver
+
+homepage = 'https://github.com/cboursnell/crb-blast'
+description = """Conditional Reciprocal Best BLAST - high confidence ortholog assignment.
+ CRB-BLAST is a novel method for finding orthologs between one set of sequences and another.
+ This is particularly useful in genome and transcriptome annotation."""
+
+toolchain = {'name': 'intel', 'version': '2017a'}
+
+dependencies = [
+    ('Ruby', rubyver, '', True),
```

Comment:
The `Ruby` built with `dummy` was required for `FPM`, but in this case we should prefer a non-dummy `Ruby` (e.g. #5573)